bin/npm-cli.js

```diff
@@ -83,7 +83,7 @@
     if (er) return errorHandler(er)
     npm.commands[npm.command](npm.argv, function (err) {
       // https://www.youtube.com/watch?v=7nfPu8qTiQU
-      if (!err && npm.config.get('ham-it-up')) {
+      if (!err && npm.config.get('ham-it-up') && !npm.config.get('json')) {
```

Comment:
Could you also check for [`parseable`](https://docs.npmjs.com/misc/config#parseable) here? I think with that, you'll be covered for output-related stuff 👍